utils/outputwriter/outputcontent.go

```diff
@@ -118,288 +157,719 @@ func footer(writer OutputWriter) string {
+		WriteContent(&contentBuilder, detailsContent)
+		return contentBuilder.String()
+	})
+}
+
+func getIssuesWithDetails(issues []formats.VulnerabilityOrViolationRow) (filter []formats.VulnerabilityOrViolationRow) {
+	for i := range issues {
+		if issues[i].JfrogResearchInformation != nil || issues[i].Summary != "" {
+			filter = append(filter, issues[i])
+		}
+	}
+	return
+}
+
+func getComponentIssueIdentifier(key, compName, version, watch string) (id string) {
+	parts := []string{}
+	if key != "" {
+		parts = append(parts, fmt.Sprintf("[ %s ]", key))
+	}
+	parts = append(parts, compName, version)
+	if watch != "" {
+		parts = append(parts, fmt.Sprintf("(%s)", watch))
+	}
+	return strings.Join(parts, " ")
+}
+
+func getScaSecurityIssueDetails(issue formats.VulnerabilityOrViolationRow, violations bool, writer OutputWriter) (content string) {
 	var contentBuilder strings.Builder
-	for _, location := range flow {
-		WriteContent(&contentBuilder, fmt.Sprintf("%s %s (at %s line %d)\n", "↘️", MarkAsQuote(location.Snippet), location.File, location.StartLine))
+	// Title
+	WriteNewLine(&contentBuilder)
+	WriteContent(&contentBuilder, writer.MarkAsTitle(fmt.Sprintf("%s Details", getIssueType(violations)), 3))
+	// Details Table
+	directComponent := []string{}
+	for _, component := range issue.ImpactedDependencyDetails.Components {
+		directComponent = append(directComponent, fmt.Sprintf("%s:%s", component.Name, component.Version))
+	}
+	noHeaderTable := NewMarkdownTable("", "")
+	if len(issue.Policies) > 0 {
+		noHeaderTable.AddRowWithCellData(NewCellData(MarkAsBold("Policies:")), NewCellData(issue.Policies...))
+	}
+	if issue.Watch != "" {
+		noHeaderTable.AddRow(MarkAsBold("Watch Name:"), issue.Watch)
+	}
+	if issue.JfrogResearchInformation != nil && issue.JfrogResearchInformation.Severity != "" {
+		severity := severityutils.Severity(issue.JfrogResearchInformation.Severity)
+		noHeaderTable.AddRow(MarkAsBold("Jfrog Research Severity:"), fmt.Sprintf("%s %s", writer.SeverityIcon(severity), severity.String()))
+	}
+	if issue.Applicable != "" {
+		noHeaderTable.AddRow(MarkAsBold("Contextual Analysis:"), issue.Applicable)
+	}
+	noHeaderTable.AddRowWithCellData(NewCellData(MarkAsBold("Direct Dependencies:")), NewCellData(directComponent...))
+	noHeaderTable.AddRow(MarkAsBold("Impacted Dependency:"), fmt.Sprintf("%s:%s", issue.ImpactedDependencyName, issue.ImpactedDependencyVersion))
+	noHeaderTable.AddRowWithCellData(NewCellData(MarkAsBold("Fixed Versions:")), NewCellData(issue.FixedVersions...))
+
+	cvss := []string{}
+	for _, cve := range issue.Cves {
+		cvss = append(cvss, cve.CvssV3)
+	}
+	noHeaderTable.AddRowWithCellData(NewCellData(MarkAsBold("CVSS V3:")), NewCellData(cvss...))
+	WriteContent(&contentBuilder, noHeaderTable.Build())
+
+	// Summary
+	summary := issue.Summary
+	if issue.JfrogResearchInformation != nil && issue.JfrogResearchInformation.Summary != "" {
+		summary = issue.JfrogResearchInformation.Summary
```

Comment:
Do we want to override the issue summary if JfrogReasearchInformation.Summary exists? or so we want to print them both?

Reply:
this was the old logic